src/common/prop.c

```diff
@@ -170,6 +170,8 @@ daos_prop_merge(daos_prop_t *old_prop, daos_prop_t *new_prop)
 			D_ASSERT(result_i < result_nr);
 			entry = &result->dpp_entries[result_i];
 			result_i++;
+		} else {
+			daos_prop_entry_free_value(entry);
```

Comment:
is it not freed at line 473 in daos_prop_entry_copy?

Reply:
Hmm. I miss this. Thanks!